config/application.rb

```diff
@@ -22,18 +21,28 @@ class Application < Rails::Application
 
 # Do not swallow errors in after_commit/after_rollback callbacks.
 
-  ActionView::Base.field_error_proc = Proc.new do |html_tag|
-    class_attr_index = html_tag.index 'class="'
+    ActionView::Base.field_error_proc = Proc.new do |html_tag|
+      class_attr_index = html_tag.index 'class="'
 
-    if class_attr_index
-      html_tag.insert class_attr_index + 7, 'error '
-    else
-      html_tag.insert html_tag.index('>'), ' class="error"'
+      if class_attr_index
+        html_tag.insert class_attr_index + 7, 'error '
+      else
+        html_tag.insert html_tag.index('>'), ' class="error"'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.